src/mono/mono/mini/mini-runtime.c

```diff
@@ -2128,7 +2145,75 @@ mono_emit_jit_dump (MonoJitInfo *jinfo, gpointer code)
 
 		record.code_index = ++code_index;
 
-		// TODO: write debugInfo and unwindInfo immediately before the JitCodeLoadRecord (while lock is held).
+		DebugEntry ent;
```

Comment:
This should follow the mono coding conventions, see the rest of the file for examples.

Reply:
done. let me know if there is anything else